config/application.rb

```diff
@@ -43,15 +43,6 @@ class Application < Rails::Application
 
     config.exceptions_app = routes
 
-    config.action_mailer.notify_settings = {
```

Comment:
Is this no longer needed? @tvararu

Reply:
Nope; we're only using Notify in production now, so no need to specify a key when developing locally